src/main/java/edu/ucsb/cs156/happiercows/entities/Commons.java

```diff
@@ -1,34 +1,54 @@
 package edu.ucsb.cs156.happiercows.entities;
 
+import java.util.Date;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+
 import lombok.Data;
 import lombok.AllArgsConstructor;
 import lombok.NoArgsConstructor;
 import lombok.Builder;
 import lombok.AccessLevel;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-
-import java.util.List;
-
-import javax.persistence.*;
-
 @Data
 @AllArgsConstructor
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 @Builder
 @Entity(name = "commons")
-public class Commons {
+public class Commons
+{
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
-  private long id;  
+  private long id;
+
   private String name;
+  private double cowPrice;
+  private double milkPrice;
+  private double startingBalance;
+  private Date startingDate;
```

Comment:
Can we use `LocalDateTime` to handle dates in entities? There's an example of this in team03. It should allow you to simplify some of the date parsing logic in the controller as well.

Reply:
Darn.

Reply:
Double-edged sword. It's awesome that the built-in APIs are so comprehensive, but it can be very overwhelming to learn all of it.

Reply:
Wow. That was painless.